backend/app/views/spree/admin/orders/index.html.erb

```diff
@@ -114,7 +114,7 @@
         <div class="col-xs-12 col-md-4">
           <div class="form-group">
             <%= label_tag :q_store_id_in, Spree.t(:store) %>
-            <%= f.select :store_id_in, Spree::Store.order(:name).pluck(:name, :id), { include_blank: true }, class: 'select2' %>
+            <%= f.select :store_id_in, Spree::Store.order('spree_stores.name').pluck(:name, :id), { include_blank: true }, class: 'select2' %>
```

Comment:
`spree_stores` -> `Spree::Store.table_name` - it's safer :)

Reply:
Good catch 👍